### PR TITLE
Ocrvs 9199

### PR DESCRIPTION
**Pull request by cibelius**
## Description

https://github.com/opencrvs/opencrvs-core/issues/9199

Port more print certificate tests to V2.

Some suites were left out, since they test the "Print in advance" functionality which is not implemented in events V2.

## Checklist

- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Comment by adskyiproger** [comment]
Your environment is deployed to https://ocrvs-9199.opencrvs.dev.

File: e2e/testcases/v2-print-certificate/birth/36.07-validate-certify-record-page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    await page.getByRole('button', { name: 'Continue' }).click()
+    await expectInUrl(
+      page,
+      `/print-certificate/${eventId}/review?templateId=v2.birth-certificate`
+    )
+  })
+
+  // @TODO: this is not implemented in events v2 yet
+  test.skip('7.3 should skip payment page if payment is 0', async () => {
+    // await page.goto(`${CLIENT_URL}/registration-home/print/1`)
```
Comment (makelicious) [inline e2e/testcases/v2-print-certificate/birth/36.07-validate-certify-record-page.spec.ts]: skip or comment out

File: e2e/testcases/v2-print-certificate/birth/data/birth-declaration.ts [hunk 2]
```diff
@@ -7,7 +7,31 @@ import { getAllLocations, getLocationIdByName } from '../../../birth/helpers'
...
+      'informant.firstname': faker.person.firstName(),
+      'informant.surname': faker.person.lastName(),
+      'informant.dob': '2008-09-12',
+      'informant.nationality': 'FAR',
+      'informant.idType': 'NATIONAL_ID',
+      'informant.nid': faker.string.numeric(10)
+    }
+  }
+
+  throw 'Unsupported informant type'
```
Comment (makelicious) [inline e2e/testcases/v2-print-certificate/birth/data/birth-declaration.ts]: error

**Review by makelicious — APPROVED** [review]
Very nice!
